internal/dialect/dialectquery/duckdb.go

```diff
@@ -0,0 +1,39 @@
+package dialectquery
+
+import "fmt"
+
+type Duckdb struct{}
+
+var _ Querier = (*Duckdb)(nil)
+
+func (s *Duckdb) CreateTable(tableName string) string {
+	q := `
+	CREATE SEQUENCE %s_id;
```

Comment:
I don't have much experience with duckdb, is this the recommended way to get an autoincrementing primary key?

Reply:
At this time I'm hardly an expert but that seemed to be the main way to achieve this. You certainly don't have an `AUTOINCREMENT` keyword on the column available.